app/services/katello/pulp3/repository/yum.rb

```diff
@@ -112,6 +112,11 @@ def copy_units(source_repository, content_unit_hrefs, dependency_solving, dest_b
           tasks
         end
 
+        def repair(repository_version_href)
+          data = PulpRpmClient::RepositoryVersion.new
+          api.repository_versions_api.repair(repository_version_href, data)
```

Comment:
out of curiosity, why do you have to pass a new/blank instance of  PulpRpmClient::RepositoryVersion  ?

Reply:
The RepositoryVersion is required, but curiously the only data element is the `base_version` value, which is optional. Without the RepositoryVersion the pulp_rpm_client throws an exception, for example `wrong number of arguments (given 1, expected 2..3)`